src/sentry/auth/helper.py

```diff
@@ -374,13 +375,13 @@ def _handle_unknown_identity(self, identity):
         if not op:
             if request.user.is_authenticated():
                 return self.respond('sentry/auth-confirm-link.html', {
-                    'identity': identity,
+                    'identity': InterfaceUser.to_python(identity),
```

Comment:
basically, this was a way to use the User interface's get_label and get_display_name methods, which makes passing different user-like objects into the templating function a lot easier.. but yes, should probably talk in person.